src/components/MainSidebar/ProgramDimensionsPanel/ProgramDimensionsFilter.js

```diff
@@ -0,0 +1,76 @@
+import i18n from '@dhis2/d2-i18n'
+import { SingleSelect, SingleSelectOption, Input } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { sGetUiInputType } from '../../../reducers/ui.js'
+import { INPUT_TYPES } from '../InputPanel/index.js'
+import styles from './ProgramDimensionsFilter.module.css'
+import { StageSelect } from './StageSelect.js'
+
+const DIMENSION_TYPES = {
```

Comment:
Unless `DIMENSION_TYPES` itself is used somewhere I'd expect these consts to be exported individually, e.g. `export const DIMENSION_TYPE_ALL` instead of being props in an object.
Also, super minor, but why the combined export at the bottom instead of just `export const` directly?

Reply:
Same for `PROGRAM_TYPES` in ProgramDimensionPanel.js

Reply:
I did this for `INPUT_TYPES`, `DIMENSION_TYPES` and `PROGRAM_TYPES` in 3124da2